docs/usage/self-hosted-experimental.md

```diff
@@ -134,6 +134,15 @@ Skip initializing `RE2` for regular expressions and instead use Node-native `Reg
 ## `RENOVATE_X_MERGE_CONFIDENCE_API_BASE_URL`
 
 If set, Renovate will query this API for Merge Confidence data.
+
+If you use Mend Renovate Enterprise and:
+
+- have a static merge confidence token that you set via `MEND_RNV_MC_TOKEN`
+- _or_ set `MEND_RNV_MC_TOKEN` to `auto`
+
+Then you must set this variable at the _server_ and the _workers_.
+
+But if you have specified the token as a [`matchConfidence`](https://docs.renovatebot.com/configuration-options/#matchconfidence) `hostRule`, you only need to set this variable at the _workers_.
 This feature is in private beta.
```

Comment:
```suggestion

This feature is in private beta.
```

This sentence needs to be its own paragraph. This sentence has no connection to the previous part about the token.